examples/graph.py

```diff
@@ -209,8 +209,30 @@ def main_shadow(self, w):
         bg = urwid.AttrMap(urwid.SolidFill("▒"), "screen edge")
         shadow = urwid.AttrMap(urwid.SolidFill(" "), "main shadow")
 
-        bg = urwid.Overlay(shadow, bg, ("fixed left", 3), ("fixed right", 1), ("fixed top", 2), ("fixed bottom", 1))
-        w = urwid.Overlay(w, bg, ("fixed left", 2), ("fixed right", 3), ("fixed top", 1), ("fixed bottom", 2))
+        bg = urwid.Overlay(
+            shadow,
+            bg,
+            urwid.LEFT,
+            urwid.RELATIVE_100,
+            urwid.TOP,
+            urwid.RELATIVE_100,
```

Comment:
it we're breaking this into multiple lines there's room to use keyword args to make the parameters easier to understand